README.md

```diff
@@ -287,47 +301,113 @@ module.exports = {
     }
     ```
+
 - **failOnNewStates:** Option to set build to failure when `new` states are found, and to disable using `new` states as a baseline. Defaults to true.
 - **alwaysAcceptBaseBranch:** Option to automatically accept `new` and `changed` states in base branch. Assumes base branch should always be correct.
 - **disableAutoSnapshots:** Option to disable initial visual snapshots automatically captured for each state. Defaults to false.
 - **disableBranchBaseline:** Option to disable independent baseline for each feature branch, and only use base branch as baseline. Must be used with `baseBranch` option. Defaults to false.
 - **disableConcurrency:** Option to disable running tests concurrently. This may be useful when test flows run concurrently may cause data conflicts or inconsistencies. Defaults to false.
 - **newSessionForEachState:** Option to start a new test session for each state. Defaults to false.
 - **failureExitCode:** The exit code to use on failure. Defaults to 1, which will fail a CI build.
-    - To NOT fail a CI build on Screener failure, set to 0. Example:
+  - To NOT fail a CI build on Screener failure, set to 0. Example:
+
     ```javascript
     failureExitCode: 0
     ```
+
 - **browsers:** Optional array of browsers for Cross Browser Testing. Each item in array is an object with `browserName` and `version` properties.
-    - `browserName` and `version` *must* match one of the supported browsers/versions in the browser table above.
-	- Example:
-	```javascript
-    browsers: [
-      {
-        browserName: 'chrome'
-      },
-      {
-        browserName: 'safari',
-        version: '11.1'
-      }
-    ]
-    ```
+  - `browserName` and `version` *must* match one of the supported browsers/versions in the browser table above.
+  - Example:
+
+    ```javascript
+      browsers: [
+        {
+          browserName: 'chrome'
+        },
+        {
+          browserName: 'safari',
+          version: '11.1'
+        }
+      ]
+      ```
+
 - **ieNativeEvents:** Option to enable native events in Internet Explorer browser. Defaults to false.
 - **sauce:** Optional Sauce Labs credentials for Cross Browser Testing.
-    - Example:
+  - Example:
+
     ```javascript
     sauce: {
       username: 'sauce_user',
       accessKey: 'sauce_access_key',
       maxConcurrent: 10, // optional available concurrency you have from Sauce Labs
       extendedDebugging: true, // optional
-      tunnelIdentifier: 'MyTunnel01' // optional
+      tunnelIdentifier: 'MyTunnel01', // optional
+      launchSauceConnect: true // optional
     }
     ```
+
 - **vsts:** Optional configuration for integrating with Visual Studio Team Services.
-    - Example:
+  - Example:
+
     ```javascript
     vsts: {
       instance: 'myproject.visualstudio.com'
     }
     ```
+
+### Sauce Connect Integration
+
+You could run your tests through sauce connect tunnel by setting the flag `launchSauceConnect: true`:
```

Comment:
"When using Sauce Labs browsers, you have the option to use the Sauce Connect tunnel by..."

"When enabled, Sauce Connect will be launched and managed by this module, and assigned a unique tunnel identifier."

Reply:
addressed